models/conv_lstm.py

```diff
@@ -25,12 +30,16 @@ def __init__(self, num_feats, conv_kernel_size, embedding_size, num_layers):
         self.pool = nn.MaxPool1d(2, 1)
 
         # detecting
-        self.lstm = nn.LSTM(embedding_size, embedding_size, num_layers, batch_first=True)
+        if cell_norm:
```

Comment:
`dropout` could be passed to lstm, granted it doesn't do anything if it's only one layer since it's only applies to layers other than the final layer.

Reply:
+1, i forgot to pass it in. i'll keep it in case we end up using n_layers=2 in the future, although it still sucks that it won't be usable for the layernorm lstm since it hasn't been implemented there. it'll probably be ok since use of layernorm doesn't help anyways.